aten/src/ATen/cuda/Exceptions.h

```diff
@@ -53,6 +56,17 @@ C10_EXPORT const char* _cublasGetErrorEnum(cublasStatus_t error);
                 " when calling `" #EXPR "`");                   \
   } while (0)
 
+#ifdef USE_ROCM
+#define TORCH_HIPBLAS_CHECK(EXPR)                               \
+  do {                                                          \
+    hipblasStatus_t __err = EXPR;                               \
+    TORCH_CHECK(__err == HIPBLAS_STATUS_SUCCESS,                \
+                "CUDA error: ",                                 \
+                " when calling `" #EXPR "`");                   \
+  } while (0)
+#endif
```

Comment:
Why this is needed? Shouldn't hipifier just replace `TORCH_CUDABLAS_CHECK` with TORCH_HIPBLAS_CHECK?